### ISSUE TITLE
Opacity error in tutorial when rendering 3D vector PDF

### ISSUE THREAD

**Issue by nojhan** [vectorgraphics/asymptote#80]
When trying to run the 3D vector render in the tutorial (Section 3.3 —Vector graphics in 3dpage —, page 55, second example, pasted below), the execution fails on the following error:
> $ asy -f pdf bug_vector-3D_opacity.asy 
> Error: /undefined in .setopacityalpha
> Operand stack:
>    0.8
> Execution stack:
>    %interp_exit   .runexec2   --nostringval--   --nostringval--   --nostringval--   2   %stopped_push   --nostringval--   --nostringval--   --nostringval--   false   1   %stopped_push   2045   1   3   %oparray_pop   2044   1   3   %oparray_pop   --nostringval--   2025   1   3   %oparray_pop   1884   1   3   %oparray_pop   --nostringval--   %errorexec_pop   .runexec2   --nostringval--   --nostringval--   --nostringval--   2   %stopped_push   --nostringval--
> Dictionary stack:
>    --dict:967/1684(ro)(G)--   --dict:0/20(G)--   --dict:82/200(L)--
> Current allocation mode is local
> Current file position is 431
> GPL Ghostscript 9.26: Unrecoverable error, exit code 1
> /usr/share/asymptote/plain_shipout.asy: 87.10: runtime: shipout failed

With code:
```
settings.prc=false;
settings.render=0;
import graph3;
size(4cm,0);
path3 myarc = rotate(18,Z) * Arc(c=O,
normal=X, v1=-Z, v2=Z, n=10);
surface backHemisphere =
surface(myarc, angle1=0,
angle2=180, c=O, axis=Z, n=10);
surface frontHemisphere = surface(myarc, angle1=180,
angle2=360, c=O, axis=Z, n=10);
draw(backHemisphere, surfacepen=material(white+opacity(0.8),
ambientpen=white), meshpen=gray(0.4));
draw(O--X, blue+linewidth(1pt));
draw(frontHemisphere, surfacepen=material(white+opacity(0.8),
ambientpen=white), meshpen=gray(0.4));
```

Asymptote 2.41-4 on Ubuntu 18.04 bionic.

**Comment by johncbowman** [comment]
Duplicate of fixed bug #77.

**Comment by nojhan** [comment]
Sorry, I relied too much on Github's search-in-title feature.
